libgalois/include/katana/PropertyGraph.h

```diff
@@ -231,8 +231,7 @@ class KATANA_EXPORT PropertyGraph {
   /// master_nodes()[this_host].empty() is true
   /// master_nodes()[host_i][x] contains LocalNodeID of masters
   //    for which host_i has a mirror
-  const std::vector<std::shared_ptr<arrow::ChunkedArray>>& master_nodes()
-      const {
+  const std::vector<std::shared_ptr<arrow::ChunkedArray>>& master_nodes() {
```

Comment:
@scober : do you happen to know who made this function non-const?

Reply:
I do not. Does git blame me?

Reply:
not sure who removed the const. Wanted to make sure you didn't have a reason in particular.